### PR TITLE
[ID] Welcome

### PR DESCRIPTION
**Pull request by troke12**
So yeah i just finished fixing sometext

### CODE REVIEW

**Comment by MegaApplePi** [comment]
Please change the title of this pull request to something more meaningful like `[id] Welcome` or `Update Welcome (id)`.

`Update id.md` makes no sense because it can mean any `id.md` file.

File: wiki/Welcome/id.md [hunk 1]
```diff
@@ -1,136 +1,141 @@
...
 
 **Catatan:** Apabila kamu tidak memiliki beatmap tutorial tersebut karena suatu alasan tertentu, kamu dapat mengunduh ulang beatmap tersebut [disini](http://osu.ppy.sh/b/22538). Setelah memainkan tutorial yang disediakan, kamu telah siap untuk mulai mengunduh beatmap-beatmap yang berbeda!
 
-Bagaimana memulai memainkan lagu-lagu kesukaanmu!
-=================================================
-
-*Penjelasan Lengkap: [Beatmaps](ID:Beatmaps "wikilink")*
+## Bagaimana memulai memainkan lagu-lagu kesukaanmu!
+_Penjelasan Lengkap: [Beatmaps][ID:Beatmaps wikilink]_
```
Comment (MegaApplePi) [inline wiki/Welcome/id.md]: no need to prefix `ID:` in the reference link, please replace line with:
```
_Penjelasan Lengkap: [Beatmaps][Beatmaps wikilink]_
```

File: wiki/Welcome/id.md [hunk 2]
```diff
@@ -1,136 +1,141 @@
-Selamat Datang
---------------
+<!-- wiki -->
+[Beatmaps wikilink]: /wiki/Beatmaps/ "Beatmaps"
+[Game_Modifiers wikilink]: /wiki/Game_Modifiers/ "Game Modifiers"
+[Multi wikilink]: /wiki/Multiplayer/ "Multi-play"
```
Comment (MegaApplePi) [inline wiki/Welcome/id.md]: this is linked incorrectly, replace line with:
```
[Multi wikilink]: /wiki/Multi/ "Multi-play"
```

File: wiki/Welcome/id.md [hunk 3]
```diff
@@ -1,136 +1,141 @@
...
 
-![osu!](Osu-boxart.png "osu!")
-
-\_\_TOC\_\_
-
-Tutorial
---------
-
+# Tutorial
 Sebelum kamu mulai bermain osu!, kamu dianjurkan untuk memainkan beatmap tutorial pertama ([Peter Lambert - osu! tutorial](http://osu.ppy.sh/s/3756)).
```
Comment (MegaApplePi) [inline wiki/Welcome/id.md]: for all links with `http://osu.ppy.sh/` please use `https://osu.ppy.sh/` (use the `https`, protocol)

**Comment by troke12** [comment]
done

**Comment by troke12** [comment]
you need another person who understand indonesian or english? hmm okay

**Comment by MegaApplePi** [comment]
We'd prefer if another person (who natively understands the edited article's language) to take a look at it before merging.

File: wiki/Welcome/id.md [hunk 4]
```diff
@@ -1,140 +1,145 @@
...
-\_\_TOC\_\_
-
-Tutorial
---------
-
-Sebelum kamu mulai bermain osu!, kamu dianjurkan untuk memainkan beatmap tutorial pertama ([Peter Lambert - osu! tutorial](http://osu.ppy.sh/s/3756)).
+# Tutorial
+Sebelum kamu mulai bermain osu!, kamu dianjurkan untuk memainkan beatmap tutorial pertama ([Peter Lambert - osu! tutorial](https://osu.ppy.sh/s/3756)).
 
 Setelah mengunduh dan menginstal osu!, buka permainan dan layar tampilan bernama "osu! update" akan muncul, kemudian tunggu hingga pembaharuan permainan selesai secara total. Ketika layar permainan mulai muncul, nanti akan ada tombol ikon osu! berukuran sangat besar di tengah-tengah layar permainan. Klik saja pada tombol tersebut dan pilih mode permainan "Solo". Kamu dapat dengan mudah mencari beatmap yang diinginkan dengan cara mengetikkan nama atau kata kunci yang ada pada nama beatmap tersebut, namun yang kamu sedang cari disaat awal mula kamu membuka permainan ini adalah beatmap "Tutorial"-mya. Beatmap ini akan mengajarkanmu fundamental-fundamental permainan osu! mode Standard seperti *Hitcircle*, *Slider*, dan 'Spinner''.
```
Comment (RaikireHiuduo) [inline wiki/Welcome/id.md]: Formatting:-
```Markdown
Setelah mengunduh dan menginstal osu!, buka permainan dan layar tampilan bernama "osu! update" akan muncul, kemudian tunggu hingga pembaharuan permainan selesai secara total. Ketika layar permainan mulai muncul, nanti akan ada tombol ikon osu! berukuran sangat besar di tengah-tengah layar permainan. Klik saja pada tombol tersebut dan pilih mode permainan "Solo". Kamu dapat dengan mudah mencari beatmap yang diinginkan dengan cara mengetikkan nama atau kata kunci yang ada pada nama beatmap tersebut, namun yang kamu sedang cari disaat awal mula kamu membuka permainan ini adalah beatmap "Tutorial"-mya. Beatmap ini akan mengajarkanmu fundamental-fundamental permainan osu! mode Standard seperti _Hitcircle_, _Slider_, dan _Spinner_.
```

File: wiki/Welcome/id.md [hunk 5]
```diff
@@ -1,140 +1,145 @@
...
+## Sumber-sumber Berguna
+Berikut adalah beberapa sumber yang dapat membantumu:-
 
--   [Latihan Awal](http://osu.ppy.sh/forum/viewtopic.php?f=14&t=5456) - Pilih beatmap untuk memulai dirimu.
--   [Pak Beatmap](http://osu.ppy.sh/p/packlist) - Dapatkan koleksi beberapa beatmap yang ditata menurut secara umum.
--   [Daftar Beatmap](http://osu.ppy.sh/p/beatmaplist) - Unduh beberapa beatmap (lagu) baru untuk bermain.
+-   [Latihan Awal](https://osu.ppy.sh/forum/viewtopic.php?f=14&t=5456) - Pilih beatmap untuk memulai dirimu.
+-   [Pak Beatmap](https://osu.ppy.sh/p/packlist) - Dapatkan koleksi beberapa beatmap yang ditata menurut secara umum.
+-   [Daftar Beatmap](https://osu.ppy.sh/p/beatmaplist) - Unduh beberapa beatmap (lagu) baru untuk bermain.
 -   [Hal yang Sering Ditanyakan](/wiki/FAQ) - Cari tahu tentang dasar dari permainan ini.
```
Comment (RaikireHiuduo) [inline wiki/Welcome/id.md]: ` -   [FAQ](/wiki/FAQ) - Cari tahu tentang dasar dari permainan ini.`

File: wiki/Welcome/id.md [hunk 6]
```diff
@@ -1,140 +1,145 @@
...
--   [osu!academy](ID:Osu!academy "wikilink") - Video yang dapat membuatmu lebih akrab dalam setiap aspek dalam osu!
--   [Tips dan Trik untuk Memulai osu!](ID:Beginner "wikilink") - Butuh cukup bantuan? Cek disini!
+-   [osu!academy](Osu!academy "wikilink") - Video yang dapat membuatmu lebih akrab dalam setiap aspek dalam osu!
+-   [Tips dan Trik untuk Memulai osu!](Beginner "wikilink") - Butuh cukup bantuan? Cek disini!
```
Comment (RaikireHiuduo) [inline wiki/Welcome/id.md]:
```Markdown
-   [osu!academy](/wiki/Announcements/osu!academy/ "osu!academy") - Video yang dapat membuatmu lebih akrab dalam setiap aspek dalam osu!
-   [Tips dan Trik untuk Memulai osu!](/wiki/Beginner "Tips dan Trik untuk Memulai osu!") - Butuh cukup bantuan? Cek disini!
```

File: wiki/Welcome/id.md [hunk 7]
```diff
@@ -1,140 +1,145 @@
...
+![osu!][osu-boxart image]
 
 Pertama-tama kami ingin memberikan sambutan dan pelukan hangat karena kamu telah bergabung dengan komunitas kami. Biar kami jelaskan sebagian hal yang cepat sebelum kamu memulai perjalananmu disini.
 
-![osu!](Osu-boxart.png "osu!")
+# Tutorial
+Sebelum kamu mulai bermain osu!, kamu dianjurkan untuk memainkan beatmap tutorial pertama ([Peter Lambert - osu! tutorial](https://osu.ppy.sh/s/3756)).
 
-\_\_TOC\_\_
+Setelah mengunduh dan menginstal osu!, buka permainan dan layar tampilan bernama "osu! update" akan muncul, kemudian tunggu hingga pembaharuan permainan selesai secara total. Ketika layar permainan mulai muncul, nanti akan ada tombol ikon osu! berukuran sangat besar di tengah-tengah layar permainan. Klik saja pada tombol tersebut dan pilih mode permainan "Solo". Kamu dapat dengan mudah mencari beatmap yang diinginkan dengan cara mengetikkan nama atau kata kunci yang ada pada nama beatmap tersebut, namun yang kamu sedang cari disaat awal mula kamu membuka permainan ini adalah beatmap "Tutorial"-nya. Beatmap ini akan mengajarkanmu fundamental-fundamental permainan osu! mode Standard seperti _Hitcircle_, _Slider_, dan _Spinner_.
```
Comment (gfjaru) [inline wiki/Welcome/id.md]: fundamental-fundamental -> dasar-dasar

File: wiki/Welcome/id.md [hunk 8]
```diff
@@ -41,33 +41,33 @@ Pertama-tama kami ingin memberikan sambutan dan pelukan hangat karena kamu telah
...
+Mungkin kamu bertanya-tanya: Apakah beatmap itu? Seperti pada permainan-permainan lainnya, beatmap memiliki tingkatan, atau terkadang tingkat kesusahan. *Nah* di osu!, kami memiliki "Beatmaps" yang jelas merupakan apa yang sebenarnya kamu ketahui, mereka adalah sebuah laagu yg memiliki ketukan (ritme). Mencari beatmap untuk lagu-lagu yang kamu inginkan itu sangat mudah! Kamu dapat menemukan setiap beatmap yang telah diajukan [disini][Beatmaps link]. Kamu dapat mencari apapun yang berhubungan dengan beatmap/lagu yang kamu cari: judul lagu, nama penyanyi, judul album, bahkan jenis lagunya! Sebagai contoh, kami akan mencoba mencari beatmap untuk lagu "Coldplay - Don't Let It Break Your Heart"
```
Comment (gfjaru) [inline wiki/Welcome/id.md]: *Mungkin kamu bertanya-tanya apa beatmap itu?

File: wiki/Welcome/id.md [hunk 9]
```diff
@@ -1,140 +1,145 @@
...
+[welcome02 image]: ./Welcome02.png "Beatmap download"
+[welcome03 image]: ./Welcome03.png "Opening the .osz file"
+[welcome04 image]: ./Welcome04.png "Beatmap in the Song Selection screen"
+[Chat_Console friendlist image]: ./Chat_Console_friendlist.png "Chat Console's friendlist"
+
+<!-- shared image -->
+[.osz icon link]: /wiki/shared/osz_icon_full.png ".osz file, clean beatmap file"
+[Accuracy results image]: /wiki/shared/Accuracy_results.png "Accuracy of player"
+
+<!-- Title -->
```
Comment (TPGPL) [inline wiki/Welcome/id.md]: L1-35: Hover texts aren't translated

**Comment by TPGPL** [comment]
Resolve conflicts

**Comment by TPGPL** [comment]
Sorry, but the Welcome page is going to be rewritten soon.

**Comment by TPGPL** [comment]
Sorry, but the Welcome page is going to be rewritten soon.

**Pull request closed without merging**